deps/build.jl

```diff
@@ -1,15 +1,37 @@
-#
-
-find_llvm() = begin
+#Returns the path as determined by llvm-config, or throws an error if LLVM
+#could not be found
+function find_llvm()
     if haskey(ENV, "TRAVIS")
         return readchomp(`which llvm-config-3.4`)
     end
     tmp = joinpath(JULIA_HOME, "llvm-config")
```

Comment:
I think since v"0.5" `llvm-config` no longer is in `JULIA_PATH`, but is installed to `joinpath(JULIA_PATH, "..", "tools")`.

See how `Cxx` locates `llvm-config` https://github.com/Keno/Cxx.jl/blob/77ae9463a5307780f5c4abf959ebb620ed17ce2e/deps/build.jl#L25